### ISSUE TITLE
Package name in CMakeLists.txt is not correct (for me)

### ISSUE THREAD

**Issue by kbabioch** [DNS-OARC/flamethrower#5]
While packaging this for Arch Linux, I've realized that the provided `CMakeLists.txt` file is not working for me out of-the-box. The offending line is this:

```
pkg_check_modules(LIBLDNS REQUIRED libldns)
```

I'm not a CMake expert, changing the `libldns` to `ldns` with some sed magic fixed the issue for me:

```
sed -i 's/libldns/ldns/' CMakeLists.txt
```

`ldns` is also the name of the package on Arch Linux, which provides the necessary include files, etc. so you probably want to add in some conditionals here to be more universal and not assume `libdns` in all cases.

**Comment by kbabioch** [comment]
This is probably already addressed with https://github.com/DNS-OARC/flamethrower/pull/3/commits/a6c927e75ff4bc23b2123c0ec5485b92d18354f4, which is not yet merged, though.

**Comment by fcelda** [comment]
Thanks for the report. I realised this problem as well when trying to build Docker containers with various base images. I just merged #3 so this should be resolved in master.
